conn.go

```diff
@@ -120,6 +128,13 @@ func NewConn(conn net.Conn, topic string, partition int) *Conn {
 	})
 }
 
+func emptyToNullable(transactionalID string) (result *string) {
+	if transactionalID != "" {
+		*result = transactionalID
```

Comment:
Isn't this dereferencing a `nil` pointer?
```suggestion
		result = &transactionalID
```

Reply:
:man_facepalming: fixed